README.md

```diff
@@ -1 +1,30 @@
-# functional-ttt
\ No newline at end of file
+# Tic-Tac-Toe
+A functionally written tic-tac-toe game using React
```

Comment:
I like that you considered the readme and added info about this project, even if its simple at the moment. The details matter. Well done :)